### PR TITLE
Added the Valid login case, invalid username case, invalid password case

### PR DESCRIPTION
**Pull request by CarlosGuadamuz0**

### CODE REVIEW

File: src/test/java/ui/orange_hrm/pages/LoginPage.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    }
+
+    /**
+     * Fills the username input with the given username.
+     *
+     * @param username the username to fill
+     * @return the login page instance
+     */
+    public LoginPage fillUsernameInput(String username) {
+        sendKeys(usernameInput, username);
```
Comment (BohdanKalep) [inline src/test/java/ui/orange_hrm/pages/LoginPage.java:61]: Before sending keys you need to check that field is visible

File: src/test/java/ui/orange_hrm/pages/LoginPage.java [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    }
+
+    /**
+     * Fills the password input with the given password.
+     *
+     * @param password the password to fill
+     * @return the login page instance
+     */
+    public LoginPage fillPasswordInput(String password) {
+        sendKeys(passwordInput, password);
```
Comment (BohdanKalep) [inline src/test/java/ui/orange_hrm/pages/LoginPage.java:73]: Before sending keys you need to check that field is visible

File: src/test/java/ui/orange_hrm/pages/LoginPage.java [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        return new DashboardPage();
+    }
+
+    /**
+     * Clicks the login button for an invalid test case.
+     *
+     * @return the login page instance
+     */
+    public LoginPage clickLoginButtonInvalidTestCase() {
+        clickElement(loginButton);
```
Comment (BohdanKalep) [inline src/test/java/ui/orange_hrm/pages/LoginPage.java:96]: Before clicking you need to check that element is visible/enabled

File: src/test/java/ui/orange_hrm/pages/LoginPage.java [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+        logInfo("Filled password input with the given password");
+        return this;
+    }
+
+    /**
+     * Clicks the login button for a valid test case and navigates to the dashboard page.
+     *
+     * @return the dashboard page instance
+     */
+    public DashboardPage clickLoginButtonValidTestCase() {
```
Comment (BohdanKalep) [inline src/test/java/ui/orange_hrm/pages/LoginPage.java:83]: Better to have this one method for valid and invalid scenarios, click login button and then verify if error message is shown or home page is opened in the new method

File: src/test/java/utils/SelenideHelper.java [hunk 5]
```diff
@@ -322,4 +323,24 @@ public static String sendKeys(SelenideElement element, String text) {
         logInfo("Send %s keys to %s element", text, element.getTagName());
         return text;
     }
+
+    /**
+     * Should be visible.
+     *
+     * @param element the element
+     */
+    public static void shouldBeVisible(SelenideElement element) {
```
Comment (BohdanKalep) [inline src/test/java/utils/SelenideHelper.java:332]: You are creating method for the existed logic. You copypasted logic of the shouldBe() method and added logger. I think its redundant

File: src/test/java/utils/SelenideHelper.java [hunk 6]
```diff
@@ -322,4 +323,24 @@ public static String sendKeys(SelenideElement element, String text) {
...
+        element.shouldBe(visible);
+        logInfo("Element %s is visible", element.getTagName());
+    }
+
+    /**
+     * Should be enabled.
+     *
+     * @param element the element
+     */
+    public static void shouldBeEnabled(SelenideElement element) {
```
Comment (BohdanKalep) [inline src/test/java/utils/SelenideHelper.java:342]: You are creating method for the existed logic. You copypasted logic of the shouldBe() method and added logger. I think its redundant

**Comment by CarlosGuadamuz0** [comment]
Hi! I solved the comments!

**Comment by BohdanKalep** [comment]
> Hi! I solved the comments!

I dont see the changes

**Comment by CarlosGuadamuz0** [comment]
Im sorry, I did not push the commit.

**Pull request merged**